lib/jackal/utils/spec/helpers.rb

```diff
@@ -85,8 +85,20 @@ def track_execution(klass)
 
 # Convenience method to check whether or not callback was executed
 
-# @param  [Smash] payload result from callback execution
+# @param payload [Smash] payload result from callback execution
 # @return [Boolean] callback execution status
 def callback_executed?(payload)
   payload.get(:executed) == true
 end
+
+# Convenience method for sending an actor a payload and waiting for result
+
+# @param actor [Carnivore::Source::Actor] actor to receive payload
+# @param payload [Smash] payload to send actor
+# @param wait_time [Integer] max time to wait for message result (default 1)
```

Comment:
kk, updated.  This is a prereq for this guy:
https://github.com/carnivore-rb/jackal-mail/pull/2